modules/retail/pages/retail-offline.adoc

```diff
@@ -0,0 +1,33 @@
+[[retail.offline]]
+= Offline Use
+
+If the {susemgr} Server is offline, you can still perform a reboot on the terminals.
+If the connection between the branch server and the terminals is lost or unstable, you can still perform an update on the terminals, using caching.
+
+[IMPORTANT]
+====
+Offline features are available only for {susemgr} Server 4.0 and above.
+You must have the Saltboot and PXE formula packages installed.
+====
+
+== Offline Terminal Reboot
+
+If the {susemgr} Server is offline, and a terminal is rebooted, it will fall back to a previously installed image.
+
+This will occur in these situations:
+* If the saltboot.py module on the {susemgr} Server has not been synchronized within a specified time (default value is 60 seconds).
```

Comment:
- we should say "saltboot formula has not started within a specified time". Checking saltboot.py module is implementation detail, useful mainly for debugging and it could be confusing if we specify it without context.
- name of the SALT_TIMEOUT variable is missing
- saltboot.py module is pushed by server and it's presence is tested on terminal